int64 id
string obj_class
time first_seen
time last_seen
time last_updated
bool stable
float64 metric
bool estimated

float64 x
float64 y

float64 delta_x
float64 delta_y

float64 width
float64 height

object_detection/BoundingBox box
object_detection/Lidar_Point lidar_point
